clevertap-core/src/main/java/com/clevertap/android/sdk/Constants.java

```diff
@@ -391,6 +397,8 @@ public interface Constants {
     HashSet<String> NONE_CRYPT_KEYS = new HashSet<>(Arrays.asList(KEY_ENCRYPTION_MIGRATION));
     HashSet<String> piiDBKeys = new HashSet<>(Arrays.asList(KEY_ENCRYPTION_NAME, KEY_ENCRYPTION_EMAIL, KEY_ENCRYPTION_IDENTITY, KEY_ENCRYPTION_PHONE));
 
+    HashSet<String> keysToSkipForUserAttributesEvaluation = new HashSet<>(Arrays.asList("cc", "tz", "Carrier"));
```

Comment:
check if there are more computed sdk attributes which we need to ignore.

Reply:
Currently only 3 which are added from `pushBasicProfile` in `EventQueueManager`